ocs_ci/ocs/resources/cloud_manager.py

```diff
@@ -97,6 +102,13 @@ def __init__(self):
         except CommandFailed:
             setattr(self, "rgw_client", None)
 
+        # set the client for STS enabled cluster
+        role_arn = get_role_arn_from_sub()
+        cred_dict["AWS"]["ROLE_ARN"] = role_arn
+        setattr(
+            self, "aws_sts_client", cloud_map["AWS_STS"](auth_dict=cred_dict["AWS"])
```

Comment:
There are two cases where this part can throw an exception and fail a non-sts-related MCG test:

1. `if config.DEPLOYMENT.get("sts_enabled") == None` --> ClusterNotInSTSModeException will be raised
2.  If for some reason creds_dict["AWS"] is missing

Please wrap this in a try-except block

Reply:
Addressed 1.  
But 2 seems to be unlikely because i dont see a reason for our CI to miss AWS credential? if its missing then that itself a problem and its better if the test fails.

Reply:
fair enough 👍